### DISCUSSION TITLE
Please allow disabling suggestions in comment bodies

### DISCUSSION THREAD

**Discussion by jbellis** [community/community#8286] [Copilot Conversations]
Copilot is bad at this, it's a distraction.

**Comment by transitive-bullshit** [comment]
Agreed. See also https://github.com/github/feedback/discussions/8062

**Comment by jamesonknutson** [comment]
@jbellis 

I got annoyed by this as well and wrote a VSCode extension that turns off Copilot's inlineSuggest setting whenever you're writing a comment (then turns it back on whenever you're not).

[You can find it here](https://marketplace.visualstudio.com/items?itemName=disable-copilot-comment-completions.disable-copilot-comment-completions), I hope you find it useful.
